File: api/src/main/java/fr/huiitre/tools/modules/riot/valorant/application/skin/view/ValorantSkinChromaView.java
package fr.huiitre.tools.modules.riot.valorant.application.skin.view;

import java.util.UUID;

public record ValorantSkinChromaView(
    UUID assetId,
    int chromaIndex,
    String name,
    String displayIconUrl,
    String fullRenderUrl,
    String swatchUrl,
    String streamedVideoUrl
) {}

File: api/Modules/Riot/Valorant/Application/Skin/Views/ValorantSkinChromaView.cs
namespace Tools.Api.Modules.Riot.Valorant.Application.Skin.Views;

public sealed record ValorantSkinChromaView(
    Guid AssetId,
    int ChromaIndex,
    string Name,
    string? DisplayIconUrl,
    string? FullRenderUrl,
    string? SwatchUrl,
    string? StreamedVideoUrl
);
